docs/contributing.md

```diff
@@ -58,7 +58,8 @@ commit. You can do that by creating a new commit on top of the initial commit
 `jj git push`
 will automatically force-push the bookmark.
 
-When your first PR has been approved, we typically give you contributor access,
+When your first PR has been approved, we invite you to the
+`jj-vcs/contributors` team to give you contributor access,
```

Comment:
You could make it a link to https://github.com/orgs/jj-vcs/teams/contributors if you think that's useful

Reply:
I don't think you can see that if you're not within the org, so probably not worth it.